components/Header.js

```diff
@@ -1,11 +1,16 @@
 import Image from "next/image";
 import styled from "styled-components";
 import menue from "../public/images/menü_btn.jpg";
+import logo from "../public/images/logoSpotifinder.png";
 
 export default function Header({ onMapShown }) {
   return (
     <>
       <StyledHeader>
+        <StyledLogo>
+          <Image src={logo} alt="Menü_Btn" width={433} height={91} />
+        </StyledLogo>
```

Comment:
no need to change it, but as a tipp for the future: You may want to change the icons and the logo to a svg. it is sharper and the size is smaller.

Reply:
**Good idea, do you may have a code example how to use a svg in an Image tag? :) **

Reply:
of course. I use it like this: 
<Image
          src={"/images/movie-flip-logo.svg"}
          alt="Movie Flip Logo"
          width={200}
          height={200}
          priority
        />